build/docker/build-image.sh

```diff
@@ -16,14 +16,17 @@ if [ ! -e "${DOCKERFILE}" ]; then
 fi
 
 TAG=${TAG:-devel}
-SRCREV=$(git rev-parse HEAD)
```

Comment:
Looks good to me, but maybe @bart0sh remembers why we had this tag in first place?

Reply:
looks good to me too. We had it to link locally build images to the current commit id. Not sure it's used anywhere else.